bbb-common-web/src/main/java/org/bigbluebutton/api/domain/Meeting.java

```diff
@@ -363,6 +365,22 @@ public String getDefaultAvatarURL() {
 		return defaultAvatarURL;
 	}
 
+	public void setWaitingPositionsInWaitingQueue(String userId, String idx) {
+		if (guestUsersWithPositionInWaitingLine.containsKey(userId)) {
+			guestUsersWithPositionInWaitingLine.replace(userId,idx);
+		} else {
+			guestUsersWithPositionInWaitingLine.put(userId,idx);
+		}

```

Comment:
What's the difference between `replace` and `put` in this context? Wouldn't a single `put` be enough to cover both cases?